master/api_service.go

```diff
@@ -5157,12 +5157,6 @@ func (m *Server) migrateMetaNodeHandler(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
-	if limit > defaultMigrateMpCnt {
```

Comment:
There should be a limit on the server side. According to experience, a concurrency of 3 is already quite a lot.

Reply:
OK, I will fix it.